src/core/services/theming/theming.js

```diff
@@ -538,6 +561,35 @@ function ThemingDirective($mdTheming, $interpolate, $log) {
   };
 }
 
+/**
+ * Special directive that will disable ALL runtime Theme style generation and DOM injection
+ *
+ * <link rel="stylesheet" href="angular-material.min.css">
+ * <link rel="stylesheet" href="angular-material.themes.css">
+ *
+ * <body md-theme-css>
+ *  ...
+ * </body>
+ *
+ * Note: Using md-themes-css directive requires the developer to load external
+ * theme stylesheets; e.g. custom themes from Material-Tools:
```

Comment:
If we're going to mention the Material-Tools here, this should probably be a link to the repo.